### PR TITLE
Type label modifications: 3.1-3.4

### PR DESCRIPTION
**Pull request by jrgissing**
type label mods, round 1: Update tutorials 3.3.1-3.3.4 to make use of type labels

### CODE REVIEW

**Comment by jrgissing** [comment]
@simongravelle I will publish a fork of [lammpstutorials-inputs](https://github.com/lammpstutorials/lammpstutorials-inputs/tree/2Aug2023) that incorporates these type label versions, so that you have full context for this PR

File: lammps-tutorials.tex [hunk 1]
```diff
@@ -1268,37 +1274,37 @@ \subsubsection{System preparation}
...
 Create a new text file called \textit{PARM.lammps} next to the \textit{systemcreation/} folder. Copy the following lines into \textit{PARM.lammps}:
 {\normalsize \begin{verbatim}
-mass 1 15.9994 # water
-mass 2 1.008 # water
-mass 3 28.990 # ion
-mass 4 35.453 # ion
-mass 5 26.9815 # wall
+mass O 15.9994
+mass H 1.008
+mass Na+ 22.990
```
Comment (jrgissing) [inline lammps-tutorials.tex:1279]: @simongravelle corrected Na mass here, please confirm

Comment (simongravelle) [reply lammps-tutorials.tex:1279]: Oh yes, well spotted, thanks !

**Comment by simongravelle** [comment]
> @simongravelle I will publish a fork of [lammpstutorials-inputs](https://github.com/lammpstutorials/lammpstutorials-inputs/tree/2Aug2023) that incorporates these type label versions, so that you have full context for this PR

OK OK, I will also give you access to the lammpstutorials-inputs repository. If you want you can create a separate branch there.

**Comment by jrgissing** [comment]
@simongravelle  before I continue, do you want to merge this PR and others that wrap lines to avoid further branch conflicts?

**Comment by simongravelle** [comment]
> @simongravelle before I continue, do you want to merge this PR and others that wrap lines to avoid further branch conflicts?

Yes, lets merge !

**Review by simongravelle — APPROVED** [review]

**Pull request merged**